.github/workflows/snap-testing.yml

```diff
@@ -56,10 +77,21 @@ jobs:
       - name: Checkout the local actions again
         uses: actions/checkout@v2
 
+      - name: Build snap
+        if: ${{ github.event_name == 'workflow_dispatch' && inputs.localBuild }}
```

Comment:
Is checking the event name necessary?

Reply:
You are right, this is unnecessary. Updated.